lib/package.json

```diff
@@ -44,26 +44,28 @@
     "luxon": "^1.0.0"
   },
   "dependencies": {
-    "@types/recompose": "^0.26.1",
-    "babel-runtime": "^6.26.0",
-    "react-text-mask": "^5.3.1",
+    "codecov": "^3.0.2",
+    "enzyme-react-adapter-future": "^1.1.3",
+    "react-event-listener": "^0.6.1",
+    "react-text-mask": "=5.4.1",
     "recompose": "^0.27.1"
   },
   "scripts": {
-    "test": "npm run test:moment && npm run test:date-fns && npm run test:luxon",
+    "test": "npm run test:date-fns && npm run test:luxon && npm run test:moment",
+    "codecov": "codecov --file=../coverage/clover.xml",
     "all-tests": "jest && npm run typescript",
-    "test:moment": "cross-env UTILS=moment && npm run all-tests",
-    "test:date-fns": "cross-env UTILS=date-fns && npm run all-tests",
-    "test:luxon": "cross-env NODE_ICU_DATA=./node_modules/full-icu UTILS=luxon && npm run all-tests",
+    "test:moment": "cross-env UTILS=moment npm run all-tests",
+    "test:date-fns": "cross-env UTILS=date-fns npm run all-tests",
+    "test:luxon": "cross-env UTILS=luxon npm run all-tests",
     "typescript": "tsc -p tsconfig.json",
     "start": "cross-env NODE_ENV=development rollup --config --watch",
     "prebuild": "rimraf build",
     "build:copy": "node copy.js",
-    "build:bundle": "cross-env NODE_ENV=production BABEL_ENV=modules rollup --config",
+    "build:bundle": "nross-env NODE_ENV=production BABEL_ENV=modules rollup --config",
```

Comment:
build is failing because of this typo - should be `cross-env`, not `nross-env` ;)

Reply:
🤭🤭 My fault! It’s a typo 🙄 I will fix that ASAP. Or feel free to fix that on develop

Reply:
Fixed by #501